lms/envs/common.py

```diff
@@ -2465,6 +2465,9 @@
 
     # Organizations App (http://github.com/edx/edx-organizations)
     'organizations',
+
+    # Added here to allow translations
+    'freetextresponse'
```

Comment:
No, the point is we shouldn't need to edit this file; we can localize the change to our file below.